nix/pkgs/mayastor/default.nix

```diff
@@ -107,6 +107,10 @@ in
       ../../../target/debug/mayastor-client
       ../../../target/debug/jsonrpc
       ../../../target/debug/kiiss
+      ../../../target/debug/node
```

Comment:
I am concerned about naming conflicts around these.

Could these be `mayastor-node`?

Reply:
I think maybe they could be renamed to something like `node-service`. A `mayastor-node` tends to indicate a node that mayastor is running on. These are control plane services.

Reply:
I would like to leave them as they are for now. We will probably be looking to move all these separate services into a single binary anyway, so this will probably change.

Reply:
Okay!